test/utils/userOp.ts

```diff
@@ -252,10 +253,10 @@ export async function fillUserOp(
       throw new Error("must have entryPoint to autofill nonce");
     const c = new Contract(
       op.sender!,
-      ["function nonce() view returns(address)"],
+      ["function nonce(uint256) view returns(uint256)"],
       provider
     );
-    op1.nonce = await c.nonce().catch(rethrow());
+    op1.nonce = await c.nonce(ACCOUNT_ABSTRACTION_FLOW).catch(rethrow());
```

Comment:
should we use nonce() method for AA flow [ which by default serves nonces[0] and name is also a standard ]
and getNonce method for forward transaction to get nonce for batchId 1 on client side.

and remove the overriding method nonce(_batchId) then it won't cause problems with ethers js as well

Reply:
@filmakarov